script/Arcueid/Day 6/06_00_ARC06_1_2.txt

```diff
@@ -242,7 +242,7 @@ Akiha stares straight into my eyes.
 [sha:6600d3bb536f9bb8f533e66cf79a47f21f8474b2]{
 -- Page 98, Offset 14164.
 -- 『絶対に誤魔化されませんから』という意思が、ありありと伝わってくる。
-It's crystal clear that she absolutely will not let herself be fooled.
+It's crystal clear she absolutely will not let herself be fooled.
```

Comment:
Feel I need another EN to weight in here, but some of these just seem off without the `that` to me

Reply:
This one reads better with the "that" but the previous ones have all been good